fe/fe-core/src/main/java/org/apache/doris/nereids/util/ExpressionUtils.java

```diff
@@ -1300,4 +1306,37 @@ public static boolean containUniqueFunctionExistMultiple(Collection<? extends Ex
         }
         return false;
     }
+
+    /**
+     * convert unnest to explode_* functions
+     */
+    public static org.apache.doris.nereids.trees.expressions.functions.Function convertUnnest(
```

Comment:
why not import `org.apache.doris.nereids.trees.expressions.functions.Function`

Reply:
added comment